### DISCUSSION TITLE
TRMM hosted costs?

### DISCUSSION THREAD

**Discussion by ghost** [amidaware/tacticalrmm#1602] [General]
Currently running self-hosted at £24 Digital Ocean droplet. Add $50 on top of that for signing and it shoots up to £60+ per month. 

What is the cost of hosting that amidaware offers for TRMM?

**Comment by ghost** [comment]
Anyone?

**Comment by silversword411** [comment]
Open a ticket with your agent count, and we'll give you a price https://docs.tacticalrmm.com/support/#commercial-support

**Reply by ghost** [reply]
Done.
